src/tree_builder/interface.rs

```diff
@@ -61,6 +61,10 @@ pub trait TreeSink {
     /// Do two handles refer to the same node?
     fn same_node(&self, x: Self::Handle, y: Self::Handle) -> bool;
 
+    /// Are two handles present in the same tree
+    /// https://html.spec.whatwg.org/multipage/infrastructure.html#home-subtree
```

Comment:
I don’t understand how this can ever be false. Could the doc-comment give an example?